test/gui/shared/scripts/helpers/api/ocis.py

```diff
@@ -0,0 +1,44 @@
+import json
+
+import helpers.api.HttpHelper as request
+import helpers.api.Provisioning as provisioning
+from helpers.api.utils import url_join
+from helpers.ConfigHelper import get_config
+
+created_groups = {}
+
+
+def get_graph_url():
+    return url_join(get_config('localBackendUrl'), 'graph', 'v1.0')
+
+
+def create_group(group_name):
+    url = url_join(get_graph_url(), 'groups')
+    body = json.dumps({"displayName": group_name})
+    response = request.post(url, body)
+    provisioning.assertHttpStatus(response, 201)
+    # save created group
+    global created_groups
+    resp_object = response.json()
+    created_groups[group_name] = {
+        "id": resp_object['id'],
+        "displayName": resp_object['displayName'],
+    }
+
+
+def delete_groups():
+    global created_groups
+    for group_name, group_info in list(created_groups.items()):
+        delete_group(group_info["id"])
+    created_groups = {}
+
+
+def delete_group(group_id):
+    global created_groups
+    url = url_join(get_graph_url(), 'groups', group_id)
+    response = request.delete(url)
+    provisioning.assertHttpStatus(response, 204)
+    for group_name, group_info in list(created_groups.items()):
+        if group_info["id"] == group_id:
+            del created_groups[group_name]
+            break
```

Comment:
we have to delete every groups from the created list
```suggestion
            del created_groups[group_name]
```